src/Workspaces/Remote/ServiceHub/UserOperationBooster.cs

```diff
@@ -31,10 +31,18 @@ public static UserOperationBooster Boost()
             {
                 s_count++;
 
-                if (s_count == 1)
+                try
                 {
-                    // boost to normal priority
-                    Process.GetCurrentProcess().PriorityClass = ProcessPriorityClass.Normal;
+                    if (s_count == 1)
+                    {
+                        // boost to normal priority
+                        Process.GetCurrentProcess().PriorityClass = ProcessPriorityClass.Normal;
+                    }
+                }
+                catch (Exception)
```

Comment:
From @Youssef1313: I don't think it's good idea to use a general exception. Specify the exact exception type being thrown.

I agree, I'm looking up the possible exception types.